docs/source/en/generation_strategies.md

```diff
@@ -173,6 +173,44 @@ your screen, one word at a time:
 An increasing sequence: one, two, three, four, five, six, seven, eight, nine, ten, eleven,
 ```
 
+
+## KV Cache Quantization
+
+The `generate()` method supports caching keys and values to enhance efficiency and avoid re-computations. However the key and value
+cache can occupy a large portion of memory, becoming a bottleneck for long-context generation, especially for Large Language Models.
+Quantizing the cache when using `generate()` can significantly reduce memory requirements at the cost of speed. 
+
```

Comment:
We need to clearly state that we currently only support the quanto backend, but are eager to support all other methods. IMO makes sense to have an issue to track this! WDYT @younesbelkada for other quantization techniques!

Reply:
added a sentences about `quanto` backend